### DISCUSSION TITLE
facing issues while building lint through command make lint

### DISCUSSION THREAD

**Discussion by nagarajtrivedi** [hyperjumptech/grule-rule-engine#199] [General]
Hi all, I am Nagaraj Trivedi. I am trying to build the go rule engine code by executing the make file.
I tried executing the code make link, but it gave following error.
It says that golint command is not found. I tried it installing using the command
sudo apt-get update -y
sudo apt-get install -y golint

But it is giving lot of error messages
eading package lists... Done
Building dependency tree       
Reading state information... Done
The following NEW packages will be installed:
  golint
0 upgraded, 1 newly installed, 0 to remove and 191 not upgraded.
Need to get 1,117 kB of archives.
After this operation, 3,694 kB of additional disk space will be used.
WARNING: The following packages cannot be authenticated!
  golint
E: There were unauthenticated packages and -y was used without --allow-unauthenticated

Let me know what could be the problem

**Comment by jinagamvasubabu** [comment]
Hmm, looks like issue with the installation of golint. can you install using `go mod` or `go install`

```go
go get -u golang.org/x/lint/golint
```

Check your system is all fine interms of gopath and gobin:
```go
go env GOPATH GOBIN
```

More detailed article here - http://networkbit.ch/golang-golint/. Let me know if it doesn't help

**Reply by nagarajtrivedi** [reply]
Hi Vasubabu, thanks for the information.
I have followed the suggestions/steps you have mentioned. I have found that GOPATH and GOBIN are properly set. But still it says
make: golint: Command not found
make: *** [Makefile:14: lint] Error 127

Thanks

**Reply by nagarajtrivedi** [reply]
Hi Vasubabu, it worked. I set the $PATH environment variable to include path of the golint. I modified it in the ~/.bashrc file and it worked.

Thanks

**Reply by nagarajtrivedi** [reply]
Hi Vasubabu, on another system I am executing
go get -u golang.org/x/lint/golint
but it is throwing error as 
go: cannot find GOROOT directory: /usr/local/opt/go/libexec

let me know what could be the problem

Thanks 
Nagaraj
